extensions/authentication-passport/spike-user-profile.md

```diff
@@ -2,21 +2,77 @@
 
 connect to story https://github.com/strongloop/loopback-next/issues/2246
 
-I picked the `authentication-passport` module to start the spike for more flexible user profile because compared with the custom authentication strategies, users have less control to the returned user when using the passport adapter. I believe if we could find a solution for the passport based strategies, applying similar approach to a custom strategy would be easy.
+I picked the `authentication-passport` module to start the spike for more
+flexible user profile because compared with the custom authentication
+strategies, users have less control to the returned user when using the passport
+adapter. I believe if we could find a solution for the passport based
+strategies, applying similar approach to a custom strategy would be easy.
 
 # Solution
 
-A converter function is introduced to be passed into the `StrategyAdapter`'s constructor. It takes in a custom user, converts it to a user profile described by `UserProfile` then returns it.
+A converter function is introduced to be passed into the `StrategyAdapter`'s
+constructor. It takes in a custom user, converts it to a user profile described
```

Comment:
It takes in a custom user, converts it to a user profile described by `UserProfile` `, and` then returns it.